tgui/packages/tgui/interfaces/SeedExtractor.tsx

```diff
@@ -437,35 +437,37 @@ export const TraitTooltip = (props) => {
     return t.path === props.path;
   });
   return (
-    <Tooltip
-      key=""
-      content={
-        <Table>
-          {!!props.grafting && (
-            <Table.Row>
-              <Table.Cell pb={1}>Graft gains the following trait:</Table.Cell>
-            </Table.Row>
-          )}
-          <Table.Row header>
-            <Table.Cell>
-              <Icon name={trait.icon} mr={1} />
-              {trait.name}
-            </Table.Cell>
-          </Table.Row>
-          {!!props.removable && (
-            <Table.Row>
-              <Table.Cell pb={1}>Removable trait.</Table.Cell>
-            </Table.Row>
-          )}
-          {!!trait.description && (
-            <Table.Row>
-              <Table.Cell>{trait.description}</Table.Cell>
+    !!trait && (
```

Comment:
prefer just making this an early return rather than wrapping the entire jsx block with it.